configure.ac

```diff
@@ -285,12 +285,14 @@ AC_CONFIG_FILES([
                  test/apps/Makefile
                  test/apps/sockaddr/Makefile
                  test/examples/Makefile
-                 test/gtest/Makefile
-                 test/gtest/ucm/test_dlopen/Makefile
                  test/mpi/Makefile
                  bindings/java/src/main/native/Makefile
                  ])
-
+AM_COND_IF([HAVE_GTEST],
```

Comment:
AS_IF(["X$enable_gtest" == "Xyes" ] , ...

Reply:
doesn't work too on distclean
original commit is just best, rolled back to it